Phonebook.Frontend/src/app/modules/profile-picture/components/profile-picture/enlarge-dialog/profile-picture-enlarge.dialog.scss

```diff
@@ -1,11 +1,12 @@
 img {
-  max-height: 75vh;
+  max-height: 60vh;
   width: auto;
 }
 
 .mat-dialog-content {
   display: flex;
   flex-flow: column;
+  overflow: hidden;
```

Comment:
I tested many images and never had that case, but in the case that the image is still too height the scroll bar won't show.

Reply:
k